src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -730,6 +732,191 @@ public static nuint RotateRight(nuint value, int offset)
 #endif
         }
 
+        /// <summary>
+        /// Accumulates the CRC (Cyclic redundancy check) checksum.
+        ///
+        /// Uses following Hardware Intrinsics, if supported:
+        /// * uint32_t __crc32b (uint32_t a, uint8_t b)
+        /// * uint32_t __crc32cb (uint32_t a, uint8_t b)
```

Comment:
This info shouldn't be in the "summary". Everything after the first sentence should be deleted.